README.md

```diff
@@ -51,15 +58,26 @@ schemas
             └── 1-0-0
 ```
 
+#### Remote schemas
+
+Your schemas might live in an iglu repository provided by an [iglu server][iglu-server]. To make
+sure your events are always validated against the latest deployed schemas, you can
+[configure](#configure) the repository. Schemas will be pulled on every startup.
+
 ## Configure
 
 Options can be set in `settings.json`, the defaults are:
 
 ```js
 {
-  "showSchemaValidation": false, // whether to turn validation on or off on startup
-  "schemaDir": "schemas/",       // folder where Snowplow schemas are situated
-  "listeningPort": 3000          // port NepSnowplow listens to
+    "showSchemaValidation": false, // whether to turn validation on or off on startup
+    "schemaDir": "schemas/",       // folder where Snowplow schemas are situated
+    "repo": {
```

Comment:
How about multiple repos? 😜

Reply:
That's a very valid point. Can extend this to an array. The rest of the code can be easily extended. 

One decision needs to be made when multiple exact same schemas exist. Schemas with the same vendor, name, format and version combination should be identical but there will be cases where they'll differ. 

Currently local always take preference over remote ones, but supporting multiple we don't want that the order matter. Or do we?